db/seeds/01_alta.rb

```diff
@@ -229,6 +229,44 @@
     end
   end
 
+  questions = [
```

Comment:
This is duplicated in both seed files and it's quite a bit of code. What do you think about either moving this to a single file that is used by both seeds, OR, even simpler, just use the Faker gem like you did in the tests? While the gem won't make proper questions and answers for pets, it is fine for dev and way less overhead to manage.

Reply:
Populated questions and answers using Faker to simplify code